tests/factories.py

```diff
@@ -71,7 +71,7 @@ class Meta:
 
     issue = faker.enum(Issue).value
     activity_form = faker.enum(ActivityForm).value
-    decision_no = f"PPP.{datetime.now().strftime('%Y')}.AC.{faker.pyint(1,500)}"
+    decision_no = f"PPP.{datetime.now().strftime('%Y')}.AC.{faker.pyint(1, 500)}"
```

Comment:
To be consistent (and `flake8` probably treats this line differently) with line 85.